ci/docker/pandas-db-test.dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python
+
+RUN apt-get update 
+RUN apt-get install -y postgresql postgresql-contrib
```

Comment:
What's the reason to install & configure postgres in this image instead of using the official postgres image?

```
services:
    postgres:
        image: postgres
        ....
```

Reply:
Not sure if the Postgres image includes Python, but even if it does longer term we may want to parametrize different versions of Python. We can parametrize the base Python image to build off of, but if we wanted to use Postgres as a base image we'd have to set up more layers to have that work

Reply:
Oh I was imagine something closer to

```
services:
  pandas-testing:
     build: pandas-container.dockerfile
     command: pytest ...
  postgres-db:
    image: postgres
    ports:
      "5432":"5432"
  mysql-db:
    image: mysql
    ports:
       ...
  s3:
    image: motoserver/moto
    ports:
       ...
```

such postgres is an independently running service that the testing service can talk to.

Reply:
Nice - didn't even know this was possible. Will test it out